CHANGELOG.md

```diff
@@ -5,6 +5,40 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## 20.1.0-rc.1 – 2024-11-14
+### Added
+- feat(polls): Allow to draft, export and import polls
+  [#13439](https://github.com/nextcloud/spreed/issues/13439)
+- feat(voice-messages): Auto play voice messages which are grouped together
+  [#13199](https://github.com/nextcloud/spreed/issues/13199)
+- feat(calls): Allow moderators to download a call participants list
+  [#13453](https://github.com/nextcloud/spreed/issues/13453)
+- feat(calls): Allow to enforce a maximum call length
+  [#13445](https://github.com/nextcloud/spreed/issues/13445)
+- feat(chat): Add option to directly download attachments
+  [Desktop #824](https://github.com/nextcloud/talk-desktop/issues/824)
```

Comment:
That's 20.0.2 and 1.0.0-rc.1?

Reply:
just want to make it prominent again for the minor feature release